content/blog/setting-bios-and-storage-controller-properties-with-redfish.md

```diff
@@ -15,88 +16,40 @@ tags:
-
-Each of them has a pending sub-zone called `Settings/` and explained in the next paragraph.
-
-## Deferred / pending high level concept description
-
-1. Modifications in these subsystems are performed using HTTP `PATCH` or HTTP `PUT` against `{{Subsystem-URI}}/`**`Settings/`**.
-2. Upon a successful transaction, the HTTP return code is `200 OK` with the associated message: `One or more properties were changed and will not take effect until the system is reset`.
-3. During the next system reset, the content of `{{Subsystem-URI}}/Settings/` is transferred one level up, in `{{Subsystem-URI}}/`. The return status of this transfer is present in `{{Subsystem-URI}}` with an associated message.
-
-The important thing to note in this flow is that the final status code and associated message of a property setting is visible **after the system reset**.
-
-## Successful example
-
-In this example, we create a Raid1 storage array of two physical disks using Postman. To achieve this goal, we have to issue a `PUT` to `{{iloURI}}/redfish/v1/Systems/1/SmartStorageConfig/Settings/` with a body (aka payload) similar to the example shown below:
-
-![Body to PUT to .../SmartStorageConfig/Settings/](/img/1-put-body.png "Body to PUT to .../SmartStorageConfig/Settings/")
-
-Upon successful completion of this `PUT` request, the HTTP status return code is `200 OK`, which means that the remote Web server understood what to do with this well-formed payload.
-
-The Body of the HTTP response contains an `error` property with a `SystemResetRequired` message. This property is a Redfish object sent by the Redfish server. It is there to give details about the next step to perform to finish the modification process.
-
-![PUT Return Status](/img/2-put-return-status.png "PUT Return Status")
-
-At this stage of the process, only the staging / "pending zone" of the Smart Array has changed and contains the `PUT` payload. We can verify this assertion by comparing the content of  `{{Subsystem-URI}}/Settings/` with the content of `{{Subsystem-URI/}}`.
-
-In the pending zone (`.../SmartStorageConfig/Settings/`) we can see the payload we sent to the Redfish server. However, in the "running zone" (`.../SmartStorageConfig/`) the `LogicalDrives` array is still empty:
-
-![GET Pending and Current LogicalDrives](/img/3-getpendinglogicadrives.bmp "GET Pending and Current LogicalDrives")
-
-It is now time to reset the server and perform a `GET` of the running zone. In the response body of this operation, the first Redfish object is a `@Redfish.Settings` collection containing a single `MessageID` mentioning `Success`. This single message is synonym of a successful transfer of the "pending zone" in to the "running zone". We will see later in this document what we get in case of an un-successful transfer.
```

Comment:
In the third sentence, change "**in to** the "running zone". " to " **into** the "running zone"."

Change the last sentence to read:
"You will see later in this document what occurs in the case of an unsuccessful transfer."